src/components/page-layout/__tests__/page-layout-test-cases.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PageLayout from '../page-layout';
+
+const testCases = {};
+
+testCases.basic = {
+  description: 'Basic',
+  component: PageLayout,
+  props: {
+    sidebarContent: <div>Some content</div>,
+    sidebarTitle: 'Some title',
+    children: <div>Doc content</div>
+  }
+};
+
+testCases.top = {
+  description: 'Custom top value',
+  component: PageLayout,
+  props: {
+    sidebarContent: <div>Some content</div>,
+    sidebarTitle: 'Some title',
+    sidebarContentStickyTop: 10,
+    children: <div>Doc content</div>
+  }
+};
+
+testCases.color = {
+  description: 'Specify background color',
+  component: PageLayout,
+  props: {
+    sidebarContent: <div>Some content</div>,
+    sidebarTitle: 'Some title',
+    sidebarColor: 'bg-white',
+    children: <div>Doc content</div>
+  }
+};
+
+export { testCases };
```

Comment:
I'm not able to verify that this component is working as expected with a narrow viewport. Do you think there's any way we can change these test cases to demonstrate that?

Reply:
Can you check out the changes I made in https://github.com/mapbox/frontend-documentation/pull/10/commits/e6530595cca2a30a562be2370968a97e4ad921ec and see if it's any clearer? I'm not sure what you're hoping to see.

Reply:
![jul-13-2018 16-48-07](https://user-images.githubusercontent.com/628431/42718297-90e68cd8-86bc-11e8-9ca4-b8fceabc5381.gif)

That's what I see on a narrow screen ^^. Does that look right? I'm not sure if it's right or not.

Reply:
Oh I see. I made a prop in https://github.com/mapbox/frontend-documentation/commit/e6530595cca2a30a562be2370968a97e4ad921ec that let's you decide if you want the sidebar content to stack or not (test case one = `false`, test case two = `true`): 

![image](https://user-images.githubusercontent.com/10479155/42718586-1d430d80-86bf-11e8-8152-515c03c85183.png)

I _think_ if the content is stacked then we want it to stick (at least in the current design that's what's happening on [this page](https://www.mapbox.com/ios-sdk/maps/overview/) on narrow screens).

Reply:
@davidtheclark I think this is the last thing to address before merging... what do you think?

Reply:
I'm not sure about it but think you could just move forward and see how things go 🤷‍♂️

Reply:
Hmmm. Ok. Once all the open PRs are merged, I'll mess around with it and see if it's going to do what I want it to do in context and then revisit this general component to see if I can make it a little clearer.